discovery-provider/src/api/v1/tracks.py

```diff
@@ -602,6 +603,35 @@ def get(self, track_id):
         stems = list(map(stem_from_track, stems))
         return success_response(stems)
 
+track_remixables_route_parser = reqparse.RequestParser()
+track_remixables_route_parser.add_argument('limit', required=False, type=int)
+@ns.route("/remixables")
```

Comment:
so this `@ns.route` thing will make this available at v1/tracks/remixables. since that's the case, we could drop the libs code in this PR and do things with the APIClient that lives in audius-client 

https://github.com/AudiusProject/audius-client/blob/6953106a16069a84726e57755965166ac98b2ab1/src/services/audius-api-client/AudiusAPIClient.ts#L47